main.cpp

```diff
@@ -16,15 +16,13 @@ video: Chapter 2 - Part 3
  
  1) Write down the names of the 6 major primitive types available in C++  here:
  
- 
- 
- 
- 
- 
- 
- 
- 
- 
+ int
+ float
+ bool
+ double
+ void
+ wchar_t 
```

Comment:
you are missing the `char` type.
add it here
declare 3 instances in `variableDeclarations()`
- [ ] revise